docs/overview.md

```diff
@@ -2,18 +2,18 @@
 
 ## ChatOps
 
-_"ChatOps is an operational paradigm where work that is already happening in the background today is brought into a common chatroom. By doing this, you are unifying the communication about what work should get done with actual history of the work being done."_ - [StackStorm](https://docs.stackstorm.com/chatops/chatops.html)
+_"ChatOps is an operational paradigm where work that is already happening in the background today is brought into a common chatroom. By doing this, you are unifying the communication about what work should get done with a actual history of the work being done."_ - [StackStorm](https://docs.stackstorm.com/chatops/chatops.html)
```

Comment:
This is still a change. 

The original sentence was `By doing this, you are unifying the communication about what work should get done with a actual history of the work being done.`